lib/msal-angular/docs/redirects.md

```diff
@@ -1,23 +1,56 @@
 # Using redirects in MSAL Angular
 
-When using redirects with MSAL, it is **mandatory** to handle redirects with either the `MsalRedirectComponent` or `handleRedirectObservable`. While we recommend `MsalRedirectComponent` as the best approach, both approaches are detailed below.
+When using redirects with MSAL, it is **mandatory** to handle redirects with either the `MsalRedirectComponent` or `handleRedirectObservable`.
 
 Note that specific guidance has been added for using MSAL Angular with Angular standalone components below.
 
-1. [`MsalRedirectComponent`](#msalredirectcomponent-a-dedicated-handleredirectobservable-component)
 1. [Subscribing to `handleRedirectObservable` manually](#subscribing-to-handleredirectobservable-manually)
+1. [`MsalRedirectComponent`](#msalredirectcomponent-a-dedicated-handleredirectobservable-component)
 1. [Redirects with standalone components](#redirects-with-standalone-components)
 
-## `MsalRedirectComponent`: A dedicated `handleRedirectObservable` component
+## Subscribing to `handleRedirectObservable` manually
+
+Many apps using Angular 17 or above will be unable to handle redirects with `MsalRedirectComponent`. If you are unable to bootstrap the `MsalRedirectComponent`, you **must** handle redirects using the `handleRedirectObservable` as follows:
```

Comment:
Do we know when an app may or may not be able to use the `MsalRedirectComponent`? Would help to provide more explicit guidance and/or deprecate/remove the `MsalRedirectComponent` altogether if no longer useful.

Reply:
We do not unfortunately

Reply:
Looks like our modules sample still works with `MsalRedirectComponent` in Angular 20 (not 17 for some reason, still unsure why). Removing this recommendation to subscribe to `handleRedirectObservable` directly.